app_dart/lib/src/datastore/cocoon_config.dart

```diff
@@ -113,6 +113,9 @@ class Config {
 
   int get maxTaskRetries => 2;
 
+  /// The number of times to retry a LUCI job on infra failures.
+  int get luciTryJobRetries => 2;
```

Comment:
nit: the variable name should mention it's for infra failure attempts, but I don't have a better name other than `maxLuciAttemptsOnInfraFailure`

Reply:
Going with luciTryInfraFailureRetries